packages/ui/.env.dist

```diff
@@ -20,3 +20,4 @@ SUPABASE_ASSET_APY_TABLE_NAME=asset-apy-development
 
 SENTRY_DSN=
 IS_SENTRY_ENABLED=
+FEATURE_DEPLOY_FLYWHEEL=
```

Comment:
This value is used in the frontend part of the code. It won't get this env variables as these are just available on netlify. You can test this locally. Try to set it. It should not be available in your localhost version either.

We need to prefix it with `NEXT_PUBLIC`
https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

Reply:
So should it be available in production only not in local?